### PR TITLE
fix: add audit fixes

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

**Review by lucas-manuel — DISMISSED** [review]

File: test/CapAutomator.t.sol [hunk 1]
```diff
@@ -554,6 +592,13 @@ contract RemoveSupplyCapConfigTests is CapAutomatorUnitTestBase {
         capAutomator.removeSupplyCapConfig(asset);
     }
 
+    function test_removeSupplyCapConfig_nonexistentConfig() public {
+        vm.prank(owner);
+        vm.expectRevert("CapAutomator/nonexistent-config");
+        capAutomator.removeSupplyCapConfig(asset);
+    }
+
```
Comment (hexonaut) [inline test/CapAutomator.t.sol:600]: Extra newline.

File: test/CapAutomator.t.sol [hunk 2]
```diff
@@ -1149,17 +1200,31 @@ contract EventTests is CapAutomatorUnitTestBase {
     }
 
     function test_RemoveSupplyCapConfig() public {
```
Comment (hexonaut) [inline test/CapAutomator.t.sol:1201]: `Remove` -> `remove`

File: test/CapAutomator.t.sol [hunk 3]
```diff
@@ -1149,17 +1200,31 @@ contract EventTests is CapAutomatorUnitTestBase {
...
+            2_000,
+            24 hours
+        );
         vm.expectEmit(address(capAutomator));
         emit RemoveSupplyCapConfig(asset);
         capAutomator.removeSupplyCapConfig(asset);
+        vm.stopPrank();
     }
 
     function test_RemoveBorrowCapConfig() public {
```
Comment (hexonaut) [inline test/CapAutomator.t.sol:1215]: `Remove` -> `remove`

File: test/CapAutomator.t.sol [hunk 4]
```diff
@@ -1149,17 +1200,31 @@ contract EventTests is CapAutomatorUnitTestBase {
...
+            1_000,
+            12 hours
+        );
         vm.expectEmit(address(capAutomator));
         emit RemoveBorrowCapConfig(asset);
         capAutomator.removeBorrowCapConfig(asset);
+        vm.stopPrank();
     }
 
     function test_UpdateSupplyCap() public {
```
Comment (hexonaut) [inline test/CapAutomator.t.sol:1229]: `Update` -> `update`. And check other places this may be incorrectly capitalized on the first word.

**Review by hexonaut — CHANGES_REQUESTED** [review]
Minor changes. Code looks good.

**Review by hexonaut — APPROVED** [review]

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**